en/docs/try-out/consent-manager.md

```diff
@@ -56,7 +45,7 @@ Follow [Configuring users and roles](../install-and-setup/configuring-users-and-
 6. Set the value for **Endpoint** as follows:
 
     ``` 
-    https://<IS_HOST>:9446/api/openbanking/consent
+    https://localhost:9443/api/openbanking/cds/backend/services
```

Comment:
no need to change this URL (unless specified for consent mgr portal)

Reply:
fixed in https://github.com/wso2/docs-ob-toolkit-cds/pull/19/commits/eeb820c41b402329079378586375b7a780a4a8aa